skillmap/src/App.tsx

```diff
@@ -75,14 +75,27 @@ class AppImpl extends React.Component<AppProps, AppState> {
     }
 
     protected handleHashChange = async (e: HashChangeEvent) => {
-        let config = await pxt.targetConfigAsync();
-        let hash = parseHash(window.location.hash || config.skillMap?.defaultPath);
-        this.fetchAndParseSkillMaps(hash.cmd as MarkdownSource, hash.arg);
-
+        await this.parseHashAsync();
         e.stopPropagation();
         e.preventDefault();
     }
 
+    protected async parseHashAsync() {
```

Comment:
small: could this be named something to indicate that it both parses the hash and grabs the skillmaps?